mlflow/tracking/client.py

```diff
@@ -3817,7 +3822,16 @@ def print_model_info(rm):
             tags: {'nlp.framework': 'Spark NLP'}
             description: This sentiment analysis model classifies the tone-happy, sad, angry.
         """
-        return self._get_registry_client().get_registered_model(name)
+        rm = self._get_registry_client().get_registered_model(name)
+
+        # Prompt should not be returned as a registered model
+        if has_prompt_tag(rm._tags):
+            raise MlflowException(
+                f"Registered Model with name={name} not found",
```

Comment:
```suggestion
                f"Registered Model with name={name!r} not found",
```

in case `name` contains a space.